app/helpers/application_helper.rb

```diff
@@ -24,6 +24,14 @@ def wordpress_path(path = '')
     Settings.wordpress.url.chomp('/') + '/' + path.sub(/^\/+/) { $1 }
   end
 
+  ##
+  # Returns frontend path with correctly joining '/'
+  # @param path String (with or without leading slash)
+  # @return String
+  def api_path(path = '')
+    Settings.api.url.chomp('/') + '/' + path.sub(/^\/+/) { $1 }
```

Comment:
Should it be `http` or `https`?  The same question goes for the link to the item in the frontend, here: https://github.com/dpla/primary-source-sets/pull/153/files#diff-94a5af5cfd56560c8341a20956c6a3f5R55  I'll have to change that too b/c it will be missing the protocol under the new config settings.